modules/core/04-channel/keeper/events.go

```diff
@@ -337,6 +337,28 @@ func emitChannelUpgradeAckEvent(ctx sdk.Context, portID string, channelID string
 	})
 }
 
+// emitChannelUpgradeConfirmEvent emits a channel upgrade confirm event
+func emitChannelUpgradeConfirmEvent(ctx sdk.Context, portID, channelID string, currentChannel types.Channel) {
```

Comment:
I don't believe we generally emit counterparty info in events, not 100% certain tho

Reply:
Like port ID and channel ID? I think we do, so should be fine.